webpack.config.js

```diff
@@ -78,6 +78,7 @@ module.exports = function(env = {}) {
             plugins: [
               [
                 'syntax-dynamic-import',
+                'transform-object-rest-spread',
```

Comment:
Was this not working before? There are object spread operators in other parts of our code no?

Reply:
I added it because there was a spread operator breaking the code from a plugin that i was linking into the project